consensus/state.go

```diff
@@ -94,8 +93,6 @@ type State struct {
 	// store blocks and commits
 	blockStore sm.BlockStore
 
-	dag format.DAGService
```

Comment:
Pls, keep DAGService in the State how it was. DAGService handles networking as well and making State rely on dag service provided by blockstore is architecturally not a good idea.

Reply:
Conceptually, I argue that we should change "move the ipfs dag api object to Blockstore" to "use the ipfs dag api object to Blockstore"

Reply:
that is now true, yes.